src/UILayer/PacketWindow.cpp

```diff
@@ -1,44 +1,120 @@
 #include <QVBoxLayout>
+#include <QPushButton>
 #include "PacketWindow.h"
+#include "../UILayer/Packet/AuxBmsTab.h"
 #include "../UILayer/Packet/BatteryFaultsTab.h"
 #include "../UILayer/Packet/BatteryTab.h"
 #include "../UILayer/Packet/DriverControlsTab.h"
 #include "../UILayer/Packet/LightsTab.h"
 #include "../UILayer/Packet/MotorTab.h"
+#include "../UILayer/Packet/MotorFaultsTab.h"
 #include "../UILayer/Packet/MpptTab.h"
 
 
-PacketWindow::PacketWindow(BatteryTab& batteryTab,
+PacketWindow::PacketWindow(AuxBmsTab& auxBmsTab,
+                           BatteryTab& batteryTab,
                            BatteryFaultsTab& batteryFaultsTab,
                            DriverControlsTab& driverControlsTab,
                            LightsTab& lightsTab,
                            MotorTab& motor0Tab,
                            MotorTab& motor1Tab,
+                           MotorFaultsTab& motor0FaultsTab,
+                           MotorFaultsTab& motor1FaultsTab,
                            MpptTab& mpptTab)
-    : batteryTab_(batteryTab)
+    : auxBmsTab_(auxBmsTab)
+    , batteryTab_(batteryTab)
     , batteryFaultsTab_(batteryFaultsTab)
     , driverControlsTab_(driverControlsTab)
     , lightsTab_(lightsTab)
     , motor0Tab_(motor0Tab)
     , motor1Tab_(motor1Tab)
+    , motor0FaultsTab_(motor0FaultsTab)
+    , motor1FaultsTab_(motor1FaultsTab)
     , mpptTab_(mpptTab)
 {
     tabs_ = new QTabWidget();
+    tabs_->addTab(&auxBmsTab_, tr("Aux BMS"));
     tabs_->addTab(&batteryTab_, tr("Battery"));
     tabs_->addTab(&batteryFaultsTab_, tr("Battery Faults"));
     tabs_->addTab(&driverControlsTab_, tr("Driver Controls"));
     tabs_->addTab(&lightsTab_, tr("Lights"));
     tabs_->addTab(&motor0Tab_, tr("Motor 0"));
     tabs_->addTab(&motor1Tab_, tr("Motor 1"));
+    tabs_->addTab(&motor0FaultsTab_, tr("Motor 0 Faults"));
+    tabs_->addTab(&motor1FaultsTab_, tr("Motor 1 Faults"));
     tabs_->addTab(&mpptTab_, tr("MPPT"));
 
     QVBoxLayout* tabsLayout = new QVBoxLayout;
+    setPacket0_ = new QPushButton("Set Packet 0", this);
+    setPacket1_ = new QPushButton("Set Packet 1", this);
+    tabsLayout->addWidget(setPacket0_);
+    tabsLayout->addWidget(setPacket1_);
     tabsLayout->addWidget(tabs_);
-    setMinimumSize(900, 700);
+    setMinimumSize(900, 800);
```

Comment:
We should set an _maximum_ size as well, it doesn't show up correctly on some computers.

Reply:
Actually, let's set the minimum size of the mainWindow instead of this one.
Let's put the minimum size to be (900, 500)!